runtime-modules/content/src/lib.rs

```diff
@@ -2427,6 +2435,8 @@ decl_module! {
             actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
             transfer_params: InitTransferParametersOf<T>,
         ) {
+            return Err(Error::<T>::FeatureNotImplemented.into());
```

Comment:
Missing `TODO` comment

Also - other extrinsics (like `destroy_nft`, `toggle_nft_limits`) are commented-out, which gave me an idea:

Why don't we just disable all extrinsics by only using:
```rust
        #[weight = 10_000_000] // TODO: adjust weight
        #[allow(unused_variables)] // TODO: Enable after carthage and remove the `allow` attribute
        pub fn initialize_channel_transfer(
```
Instead of commenting out the body of the method and prefixing with parameters with `_`.

What do you think?

Reply:
Even better approach would be filtering calls in runtime config, like https://github.com/Joystream/joystream/pull/3986
We can use cargo `features` to have different filtering for production and tests/benchmarking

Reply:
I will proceed with the recommended approach after #3986 is merged, so that I can leverage the provided trait.
For the scope of this PR I used Leszek's suggestion